snmp/distro

```diff
@@ -19,23 +19,25 @@ elif [ "${OS}" = "Linux" ] ; then
   if [ -f /etc/fedora-release ]; then
     DIST=$(cat /etc/fedora-release | awk '{print $1}')
     REV=`cat /etc/fedora-release | sed s/.*release\ // | sed s/\ .*//`
-        
+
   elif [ -f /etc/redhat-release ] ; then
     DIST=$(cat /etc/redhat-release | awk '{print $1}')
+    IGNORE_LSB=1
+    PSEUDONAME=`cat /etc/redhat-release | sed s/.*\(// | sed s/\)//`
+    REV=`cat /etc/redhat-release | sed s/.*release\ // | sed s/\ .*//`
+
     if [ "${DIST}" = "CentOS" ]; then
       DIST="CentOS"
     elif [ "${DIST}" = "Mandriva" ]; then
       DIST="Mandriva"
       PSEUDONAME=`cat /etc/mandriva-release | sed s/.*\(// | sed s/\)//`
       REV=`cat /etc/mandriva-release | sed s/.*release\ // | sed s/\ .*//`
-    elif [ -f /etc/oracle-release ]; then
-      DIST="Oracle"
     else
-      DIST="RedHat"
+      DIST="RedHat ${PSEUDONAME}"
```

Comment:
This will change the distro and potentially mean peoples alerts will trigger. It will either need a notification going out or this change reverting so it's just RedHat.

Reply:
The script isn't working to match Red Hat at this time. With the script like the original, the LSB section always replaces the Red Hat section and the returned informations by LSB are not matched by LibreNMS.

The final $OSSTR is something like RedHatEnterpriseLinux and it makes it match nothing. The name must be RedHat<space>Something to match the OS detection.

Reply:
But is RedHat as the distro not enough? That's all we do for other distros like Ubuntu, CentOS, etc afaik